src/main/java/com/uid2/admin/store/factory/KeysetKeyStoreFactory.java

```diff
@@ -22,20 +22,23 @@ public class KeysetKeyStoreFactory implements StoreFactory<Collection<KeysetKey>
     private final Clock clock;
     private final FileManager fileManager;
     private final RotatingKeysetKeyStore globalReader;
+    private final boolean enableKeyset;
 
     public KeysetKeyStoreFactory(
             ICloudStorage fileStreamProvider,
             CloudPath rootMetadataPath,
             VersionGenerator versionGenerator,
             Clock clock,
-            FileManager fileManager) {
+            FileManager fileManager,
+            boolean enableKeyset) {
         this.fileStreamProvider = fileStreamProvider;
```

Comment:
can we remove all these `enableKeyset` members and parameters and instead have a public static somewhere which returns `config.getBoolean("enable_keysets")` ?

Reply:
I am not sure I  understand why this is better? Or how to implement a static that gets a config value.

Reply:
As discussed, if it's easy to do let's make a singleton so we can get config from anywhere (similar to uid2-core). Otherwise let's leave this for now.